build_tools/benchmarks/common/benchmark_definition.py

```diff
@@ -19,14 +19,16 @@
 from enum import Enum
 from typing import Any, Dict, Optional, Sequence, Tuple
 
-# A map from CPU ABI to IREE's benchmark target architecture.
-CPU_ABI_TO_TARGET_ARCH_MAP = {
+from e2e_test_framework.definitions import common_definitions
+
+# A map from CPU ABI to IREE's legacy benchmark target architecture.
```

Comment:
Which one is the legacy name here? Should this be `CPU_ABI_TO_LEGACY_TARGET_ARCH_MAP`? The current name I parsed as `((Legacy CPU ABI) -> (Target Arch)) Map`, but I guess you mean `Legacy ((CPU ABI) -> (Target Arch)) Map` and my suggested renaming would be `((CPU ABI) -> (Legacy Target Arch)) Map`. Part of the reason it's confusing is that the comment doesn't quite match. The keys differ between the legacy map and the new map, e.g. `x86_64-CascadeLake` -> `x86_64-cascadelake`. From the lookup function it seems like the keys should always be lowercase? Is this a bug?